Riot/Modules/Room/ParticipantsInviteModal/RoomParticipantsInviteModalCoordinatorBridgePresenter.swift

```diff
@@ -0,0 +1,214 @@
+                                       detail: VectorL10n.roomInviteToRoomOptionDetail(spaceName),
+                                       image: Asset.Images.addParticipants.image, room: room,
+                                       accessoryImage: Asset.Images.chevron.image)
+        ]
+        
+        let coordinator = OptionListCoordinator(parameters: OptionListCoordinatorParameters(title: VectorL10n.roomIntroCellAddParticipantsAction, options: self.roomOptions, navigationRouter: self.navigationRouter))
+        coordinator.delegate = self
+        coordinator.start()
+        
+        self.optionListCoordinator = coordinator
+    }
+    
+    private func pushContactsPicker(for room: MXRoom) {
+        guard let session = self.session else {
+            MXLog.error("[RoomParticipantsInviteCoordinatorBridgePresenter] pushContactsPicker: nil session found")
+            return
+        }
+        
+        canInvite(to: room) { [weak self] canInvite in
+            guard let self = self else { return }
+            
+            guard canInvite else {
+                let message = room.summary?.roomType == .space ? VectorL10n.spaceInviteNotEnoughPermission : VectorL10n.roomInviteNotEnoughPermission
+                let alert = UIAlertController(title: VectorL10n.spacesInvitePeople, message: message, preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: VectorL10n.ok, style: .default, handler: nil))
+                self.navigationRouter?.present(alert, animated: true)
+                return
+            }
+            
+            let coordinator = ContactsPickerCoordinator(session: session,
+                                                        room: room,
+                                                        currentSearchText: self.currentSearchText,
+                                                        actualParticipants: self.actualParticipants,
+                                                        invitedParticipants: self.invitedParticipants,
+                                                        userParticipant: self.userParticipant,
+                                                        navigationRouter: self.navigationRouter)
+            coordinator.delegate = self
+            coordinator.start()
+            
+            self.contactPickerCoordinator = coordinator
+        }
+    }
+    
+    private func canInvite(to room: MXRoom, completion: @escaping (Bool) -> Void) {
+        guard let userId = self.session?.myUserId else {
+            MXLog.error("[RoomParticipantsInviteCoordinatorBridgePresenter] canInvite: userId not found")
+            completion(false)
+            return
+        }
+
+        room.state { roomState in
+            guard let powerLevels = roomState?.powerLevels else {
+                MXLog.error("[RoomParticipantsInviteCoordinatorBridgePresenter] canInvite: powerLevels room found")
```

Comment:
Same as before but I see the format from above :)
```suggestion
                MXLog.error("[RoomParticipantsInviteCoordinatorBridgePresenter] canInvite: room powerLevels not found")
```